test/bin/common_versions.sh

```diff
@@ -136,3 +138,7 @@ export BREW_Y2_RELEASE_VERSION
 export BREW_RC_RELEASE_VERSION
 export BREW_EC_RELEASE_VERSION
 export BREW_NIGHTLY_RELEASE_VERSION
+
+# Get the release type from most recent release in test/releases.yaml to be used in ostree commit names and bootc image names
+LATEST_RELEASE_TYPE="$(awk '/^  [^[:space:]]/{f=1;next} f&&/release_type:/{print $2;exit}' "${SCRIPTDIR}/../releases.yaml")"
```

Comment:
Maybe we should install `yq`?
`test/bin/manage_hypervisor_config.sh` is already using `yq`, although I can't find how it gets installed to system/user-wide path (compared to _output/bin/), but I think installing it to /usr/bin on hypervisor is acceptable